### PR TITLE
Player window V2

### PR DESCRIPTION
**Pull request by danielravina**
- Let the video load 144p quality initially
- Point the production player to a new v2 (already deployed, see [docs](https://headsetapp.co/docs/player-window))
- Change the dimensions of the player to fit the new design
- remove auto minimize

Added benefits of the style changes:
Users don't need to enlarge the player if they want to change the video quality/speed.

### CODE REVIEW

File: darwin/index.js [hunk 1]
```diff
@@ -60,22 +61,19 @@ const start = () => {
...
     player = new BrowserWindow({
-      width: 285,
-      height: 440,
-      resizable: true,
+      width: 427,
+      height: 300,
+      minWidth: 427,
+      minHeight: 300,
       title: 'Headset - Player',
-      maximizable: true,
```
Comment (danielravina) [inline darwin/index.js]: `true` is the default. no need for this attribute

File: shared/player/index.html [hunk 2]
```diff
@@ -4,19 +4,14 @@
...
-  <p class>
-    This is where the music comes from. Due to a restriction in YouTube's policy,
-    we can't separate the audio from the video and the player must be visible at all times.
-  </p>
   <div class="player-wrapper">
     <div id="ytplayer"></div>
-    <div class="playBtn"></div>
   </div>
-  <p>To save you some bandwidth, we select the lowest possible video quality, Don't worry, this doesn't affect the audio in any way. Enjoy!</p>
+  <div class="info">
```
Comment (m1guelpf) [inline shared/player/index.html]: Why do we need this changes here? Aren't we using the URL?

**Comment by m1guelpf** [comment]
@danielravina Great work on this, the new design looks very clean!

File: shared/player/index.html (hunk 2 above)
Comment (danielravina) [reply shared/player/index.html]: This code is eventually deployed to [here](https://danielravina.github.io/headset/player-v2/). This is the source of truth.

**Comment by danielravina** [comment]
1. I don't think that it's too much of a concern. The error doesn't affect the app in any way. it's just showing "video can't play". If we see questions come up after this, we will consider a different approach. An introductory video/easter egg is a nice idea. Once I make one (probably before V2), I'll add it there!
2. This is a problem regardless, shouldn't be addressed in the PR anyway. 
3. I will add that!

Thanks for the review @m1guelpf and @fcastilloec!

**Comment by m1guelpf** [comment]
@danielravina Can I suggest "Never Gonna Give You Up" as easter egg?

**Comment by danielravina** [comment]
giphy

**Pull request merged**